.github/workflows/ci.yml

```diff
@@ -30,18 +30,9 @@ jobs:
       - name: Clone This Repo
         uses: actions/checkout@v3
       - name: Link Verification
-        uses: FreeRTOS/CI-CD-Github-Actions/link-verifier@v2
-
-  url-check:
```

Comment:
Removing the kernel's custom script to instead use the CI-CD action that was already added and been used.